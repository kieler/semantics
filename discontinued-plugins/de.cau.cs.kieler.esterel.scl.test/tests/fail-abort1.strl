module test_abort1:
output A;
output B;
output C;
input D;
output E;
output F;
input G;
output H;
output I;
input J;
output K;
output L;

abort
  emit A;
  pause;
  emit B;
  pause
  emit C
when
  case immediate D do
    emit E;
    pause;
    emit F
  case G do
    emit H;
    pause;
    emit I
  case 5 J do
    emit K;
    pause;
    emit L
end abort

end module
