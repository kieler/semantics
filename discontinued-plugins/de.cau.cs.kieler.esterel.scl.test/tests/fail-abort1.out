line 20:3: unexpected token: emit
